Makefile

```diff
@@ -89,6 +90,7 @@ artifacts-misc:
 	# $(TAR_C) -f _artifacts/repro-get-$(VERSION).go-mod-vendor.tar go.mod go.sum vendor
 	# touch -d @$(SOURCE_DATE_EPOCH) _artifacts/repro-get-$(VERSION).go-mod-vendor.*
 	echo $(VERSION) >_artifacts/VERSION
+	echo "$(GO_VERSION)" >_artifacts/GO_VERSION
```

Comment:
Go version isn't enough, as the build depends on UPX version etc.

So I'm not inclined to just print the Go version here.

Reply:
(The `_artifacts/VERSION` file has a different story; it solely exists for resolving the latest version by `curl -fsSL  https://github.com/reproducible-containers/repro-get/releases/latest/download/VERSION`)

Reply:
Actually I was referring to the Go builds only; since the artifacts can be reproduced with the provided Dockerfile, I updated the readme.